walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/index.html

```diff
@@ -10,7 +10,7 @@
     <div id="content">
       <h1>Random Quotes</h1>
       <p>Click here to see a random quote:</p>
-      <button onclick="getRandomQuote()">Fetch a quote!</button>
+      <button onclick="getRandomQuoteUsingArrowFunctions()">Fetch a quote!</button>
```

Comment:
This function is undefined, added to commit by accident?

Reply:
The walk through wanted us to change it just to see that it also worked with that function. the function exists in a diferent directory in a file also called script.js. I wasn't sure if we were supposed to commit that or not. I assume for the future to just commit and push things pertaining to the portfolio. How do I go about uncommiting this change then?

Reply:
I don't think merging this with the master will be a problem because it has nothing to do with my code for the portfolio and the function exists in the file through the path walkthroughs/week-3-server/examples/random-quotes/src/main/webapp/script.js